www/class/centreon.class.php

```diff
@@ -172,9 +171,6 @@ public function creatModuleList()
             if (is_dir("./modules/" . $result["name"] . "/restart_pollers/")) {
                 $this->modules[$result["name"]]["restart"] = true;
             }
```

Comment:
those lines could me removed too ?

Reply:
They didn't get mentionned in the ticket but i'll remove them :)